sandbox-web/src/main/java/com/exadel/sandbox/team5/CompanyRestController.java

```diff
@@ -28,7 +32,11 @@ public CompanyDto getCompany(@PathVariable Long id) {
     }
 
     @PostMapping
-    public CompanyDto save(@RequestBody CompanyDto company) {
+    public CompanyDto save(@RequestBody CompanyDto company, MultipartFile file) throws IOException {
+        ImageDto image = new ImageDto();
+        image.setContent(file.getInputStream());
+        image.setContentType(file.getContentType());
```

Comment:
add content type validation: 
```
contentType.equals("image/png")
                || contentType.equals("image/jpg")
                || contentType.equals("image/jpeg")
```

Reply:
if contentType doesn't equals these conditions, we throw exception?

Reply:
Yes

Reply:
done